_submissions/round-12/0/2015-02-07-about-JavierGarcia.md

```diff
@@ -0,0 +1,14 @@
+#Javier García Algarra
```

Comment:
Please remove this line - in order for Jekyll to recognize the header, `---` *must* be the very first line.